reconcile/utils/terrascript_aws_client.py

```diff
@@ -5469,9 +5469,13 @@ def populate_tf_resource_asg(self, spec) -> None:
                 override
             )
 
-        asg_value["tag"] = [
+        tags = [
             {"key": k, "value": v, "propagate_at_launch": True} for k, v in tags.items()
         ]
+        if self.versions.get(account).startswith("3"):
+            asg_value["tags"] = tags
+        else:
+            asg_value["tag"] = tags
```

Comment:
I thought both versions support `tag` in the same format, no?

Reply:
both are supported, but i'm trying to perform a migration that has `No changes. Your infrastructure matches the configuration.` in every step of the way